app/util/appUtil.js

```diff
@@ -337,14 +337,35 @@ async function getItem(tokenId) {
 
   if (tokenId) {
     await api.isReady
-    const item = await api.query.simpleNftModule.tokensById(tokenId)
+    const item = (await api.query.simpleNftModule.tokensById(tokenId)).toJSON()
+    const timestamp = await getTimestamp(item.created_at)
 
-    response = item.toJSON()
+    response = { ...item, timestamp }
```

Comment:
I like that you using spread, really cool tool however in this instance we are mutatiing response, why we can not just ```js
return. { ...item, timestamp };
```

In each session we could return unless other way would make everything complicated, sorry did not look at the whole file